docs/mobile-apps/features/fast-booted-simulators.md

```diff
@@ -0,0 +1,107 @@
+---
+id: fast-booted-simulators
+title: Fast Booted iOS Simulators
+sidebar_label: Fast Booted iOS Simulators
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><small><span className="sauceDBlue">Beta</span></small></p>
+
+The speed of the Simulator is a crucial factor that impacts the development workflow of iOS app developers. Fast booted iOS Simulators offer significant advantages over our default Simulators, as they reduce the time it takes to launch the Simulator to test apps. With fast booted iOS Simulators, you can quickly switch between two iOS versions and test your app in four configurations to ensure the compatibility and performance of your app. 
+
+Sauce Labs supports the following configurations: 
+
+- iPhone 14 Simulator with the latest iOS 16.x version (current_major).
+- iPhone 13 Simulator with the latest iOS 15.x version (previous_major).
+- iPad (10th generation) Simulator with the latest iOS 16.x version (current_major).
+- iPad (9th generation) Simulator with the latest iOS 15.x version (previous_major).
+
+:::note
+By using the latest/current_major version, you can test your apps in the most up-to-date environment, while the previous_major version provides the environment for testing apps on the previous iOS version.
+:::
+
+These Simulator configurations can be used separately for automated tests, allowing you to target specific devices and iOS versions for testing. This flexibility can help you optimize the testing process and improve the compatibility and performance of your apps.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- A native iOS, or iPadOS mobile app.
+
+## Accessing Fast Booted iOS Simulator for Live Testing
+
+To access the Fast Booted iOS Simulator follow the steps below, for this example we will use the iPhone 14 with the latest iOS 16.x version (current_major):
+
+1. On Sauce Labs click **Live** > **Cross Browser** > **Mobile Virtual**.
+<img src={useBaseUrl('img/mobile-apps/fast-sim-1.png')} alt="fast booted menu" width="750"/>
+
+2. Select “Apple” as the Manufacturer.
+3. If you want to use the “current_major” version, then use these values :
+- Select **iPhone 14 Simulator** for the Device.
+- Select the latest available version of iOS.
+
+
+:::note
+If you want to use the “previous_major” version, then use these values:
+- Select “iPhone 13 Simulator” for the Device
+- Select “iOS 15.4” for the OS Version
+:::
+
+## Accessing Fast Booted iOS Simulator for Automated Testing with Appium
+
+To use the fast-booted Simulators, specify the following device configuration capabilities:
+
+For iPhone:
```

Comment:
Can we use the tabs as shown here https://docs.saucelabs.com/mobile-apps/automated-testing/appium/appium-2-migration/#android-emulators-and-ios-simulators and then only for the iOS sims?